types/workbox-sw/index.d.ts

```diff
@@ -0,0 +1,1426 @@
+// Type definitions for workbox-sw 3.2
+// Project: https://github.com/GoogleChrome/workbox
+// Definitions by: Frederik Wessberg <https://github.com/wessberg>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+declare module "workbox-sw" {
```

Comment:
`declare module "workbox-sw"` doesn't add anything, so you can just remove it and put its contents on the top level.